configure

```diff
@@ -157,7 +168,7 @@ check_bpf_use_errno()
 int dummy(void *ctx) { return 0; }
 EOF
 
-    compile_err=$($CLANG -target bpf -c $TMPDIR/bpf_use_errno_test.c 2>&1)
+    compile_err=$($CLANG -target bpf -I/usr/include/$($CC -print-multiarch) -c $TMPDIR/bpf_use_errno_test.c 2>&1)
```

Comment:
Couldn't this just reuse `$ARCH_INCLUDES`? Just make it a global var in the configure script...